dietpi/dietpi-software

```diff
@@ -12318,6 +12324,40 @@ _EOF_
 			Download_Install "$(curl -sSfL 'https://api.github.com/repos/rclone/rclone/releases/latest' | mawk -F\" "/\"browser_download_url\": .*\/rclone-v[^\"\/]*linux-$arch.deb\"/{print \$4}")"
 
 		fi
+		
+		software_id=206 # Homer
+		if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == 1 )); then
+		
+		Banner_Installing
+		
+		# installing nginx and ensuring it starts at boot
+		sudo apt install nginx -y
+		sudo systemctl enable nginx
+		
+		# installing homer
+		wget "https://github.com/bastienwirtz/homer/releases/latest/download/homer.zip" 
+		sudo unzip homer.zip -d /mnt/dietpi_userdata/Homer/
+		sudo cp /usr/share/nginx/homer/assets/config.yml.dist /mnt/dietpi_userdata/Homer/assets/config.yml
+		
+		#create nginx config - echo line by line to conf
```

Comment:
Are you planning to add `Apache` as well as `Lighttpd`? Because we offer all 3 web server stacks.

Reply:
i've mainly only used nginx in the past, not too familar with apache and lighttpd

Reply:
time to discover new things 😜